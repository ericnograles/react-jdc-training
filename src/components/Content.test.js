import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Content from './Content';

describe('<Content>', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Content />, div);
  });

  it('properly shows given text', () => {
    const output = shallow(<Content text="Unit Tests" />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});

