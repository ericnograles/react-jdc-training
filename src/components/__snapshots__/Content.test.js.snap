// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content> properly shows given text 1`] = `
<p
  className="App-intro"
>
  Unit Tests
</p>
`;
